### DISCUSSION TITLE
hex logo?

### DISCUSSION THREAD

**Discussion by maelle** [cynkra/fledge#518] [Ideas]
With such a name it'd be nice to have a logo with a bird (and I was thinking it'd make the website look nicer).

**Comment by maelle** [comment]
and in the meantime, is there a hex logo for cynkra in general?

**Comment by maelle** [comment]
@krlmlr as frivolous as a hex logo is, it'd be nice for promotion efforts. :smile_cat:

**Comment by krlmlr** [comment]
One day.

cynkra has a logo, it could fit in a hex shape.
